roles/tailscale/templates/iiab-vpn

```diff
@@ -9,6 +9,7 @@
 VPN_URL=https://iiab.net
 VPN_KEY="$1"
 
+systemctl start tailscaled
```

Comment:
If disabling Tailscale's during iiab-vpn-off, we'd need to fully reverse that here:

```suggestion
systemctl enable --now tailscaled
```

Reply:
Ok I'll go with the user is supplying the VPN_KEY= which I presume only the user knows so why leave the service running when there is user input needed? Where else would a user stash the needed key?

Reply:
https://github.com/iiab/iiab/issues/4058#issuecomment-3222172316

Reply:
That's a good reminder that we might want to run `tailscaled` (the Tailscale daemon) with `--no-logs-no-support` to disable telemetry:

https://tailscale.com/kb/1011/log-mesh-traffic#opting-out-of-client-logging